cmd/awsDataGather/main_test.go

```diff
@@ -10,12 +10,9 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
 var _ = Describe("AWS Data Gathering behavior", func() {
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Why is this removed?

Reply:
It is extra code, we are not logging anything here

Reply:
Can we do this change in another PR?